src/main/resources/logback.xml

```diff
@@ -26,18 +26,20 @@
 
     <appender name="SYSLOG" class="ch.qos.logback.classic.net.SyslogAppender">
         <syslogHost>localhost</syslogHost>
-        <facility>AUTH</facility>
-        <suffixPattern>[%thread] %logger %msg</suffixPattern>
+        <facility>logaudit</facility>
+        <suffixPattern>[%level] [%d{HH:mm:ss.SSS}] [%thread] %logger{36} - %msg%n</suffixPattern>
```

Comment:
Take a look at http://www.w3.org/TR/WD-logfile (which Khalid found) ... would be nice to normalize on that format.